OCast/src/test/java/org/ocast/dial/DialClientTest.kt

```diff
@@ -0,0 +1,501 @@
+/*
+ * Copyright 2019 Orange
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ocast.dial
+
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.SocketPolicy
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.ocast.common.HttpClientTest
+import org.ocast.common.SynchronizedFunction1
+import org.ocast.common.removeXMLElement
+import java.net.URI
+import java.net.URL
+import java.util.concurrent.TimeUnit
+import javax.xml.ws.http.HTTPException
+
+/**
+ * Unit tests for the [DialClient] class.
+ */
+internal class DialClientTest : HttpClientTest() {
```

Comment:
Android Studio does not run tests in InnerClass, use (for exemple) external lib like :

```
@RunWith(NestedRunner::class)
internal class DialClientTest : HttpClientTest() {

testImplementation “com.nitorcreations:junit-runners:1.2”
```

Reply:
Fixed.
Also fixed UpnpClientTest and SsdpMessageTest.